lib/python/rosie/vc.py

```diff
@@ -464,6 +606,47 @@ def create(argv):
             info_config = rose.config.load(info_file.name)
         finally:
             os.unlink(info_file.name)
+        new_id = ""
+        try:
+            info_config_new, report_error = client.create(info_config,
+                                                          from_id,
+                                                          opts.prefix,
+                                                          opts.meta_suite_mode,
+                                                          new_id, mode_cli=True)
+        except (RosePopenError, SuiteInfoFieldError,
+                SuiteIdOverflowError) as e:
+            report(e)
+            sys.exit(1)
+        while report_error is True:
```

Comment:
`report_error` sounds like a name for a function or method. Can we have a name that sounds more like a boolean?